mne/preprocessing/xdawn.py

```diff
@@ -188,7 +297,7 @@ class Xdawn(TransformerMixin, ContainsMixin):
     ``patterns_`` : dict of ndarray
         If fit, the Xdawn patterns used to restore M/EEG signals for each event
         type, else empty.
-    ``evokeds_`` : dict of evoked instance
+    ``evokeds_`` : dict of ndarray
```

Comment:
Okay I have changed it.
* `_evokeds` holds the data part used in the internal functions.
* `_get_evokeds` returns the dict of `EvokedArray` instances.